APMrover2/Rover.cpp

```diff
@@ -27,6 +27,7 @@ Rover::Rover(void) :
     channel_steer(nullptr),
     channel_throttle(nullptr),
     channel_aux(nullptr),
+    channel_lateral(nullptr),
```

Comment:
This just wastes flash space; these are zeroed through dint of being in a static object.  You're just doing what the other channels do, so *shrug*.